inc/namespace.php

```diff
@@ -90,6 +107,41 @@ function enqueue_scripts() {
 	$wp_scripts->add_data( 'altis-experiments', 'async', true );
 }
 
+/**
+ * Registers a new conversion goal for front end tracking.
+ *
+ * @param string $name A reference name for the goal.
+ * @param string $label A human readable label for the goal.
+ * @param string $event The JS event to trigger on.
+ * @param string|null $selector An optional CSS selector to scope the event to.
+ * @return void
+ */
+function register_goal( string $name, string $label, string $event, ?string $selector = null, ?string $closest = null ) : void {
```

Comment:
`?string` === 🤯

Reply:
Heh, yeah this is a nullable type, so a string or null is valid.